### PR TITLE
Better error handling

### PR DESCRIPTION
**Pull request by szymon-owczarzak**

### CODE REVIEW

File: cogboard-webapp/src/components/Widget/styled.js [hunk 1]
```diff
@@ -51,6 +51,17 @@ StyledCard.propTypes = {
...
+  const { errorMessage, errorCause } = props;
+
+  return (
+  <>
+    <Typography paragraph>{errorMessage}</Typography>
+    <Typography variant="caption" paragraph>{errorCause}</Typography>
+  </>
+  );
+};
+
```
Comment (Cich0sza) [inline cogboard-webapp/src/components/Widget/styled.js]: In my opinion, this component should be moved to a separate file. You may also extract props inside the parentheses. Like this `ErrorMessage = ({ errorMessage, errorCause }) => (<>...</>)`

Comment (goeson00) [reply cogboard-webapp/src/components/Widget/styled.js]: Each component should has it's own file. The exception to this are small styled components.

File: cogboard-webapp/src/components/Widget/index.js [hunk 2]
```diff
@@ -138,7 +140,10 @@ const Widget = ({ id, index }) => {
           }
         />
         <StyledCardContent>
-          {!disabled ? <WidgetContent id={id} type={type} content={content} /> : 'Disabled'}
+          {content && content.errorMessage && content.errorMessage !== ''
```
Comment (goeson00) [inline cogboard-webapp/src/components/Widget/index.js]: I suggest moving this to helper function called renderCardContent taking content and disable as arguments. This component is getting more and more bloated. I plan to do some serious refactor.
I also think content.errorMessage !== '' is redundant to content.errorMessage which will coerce to false if it is an empty string.

File: cogboard-webapp/src/components/widgets/types/AemHealthcheckWidget/index.js [hunk 3]
```diff
@@ -1,21 +1,13 @@
...
-  const { healthChecks, errorMessage } = props;
-
-  if (errorMessage) {
-    return (
-      <Typography variant="h5">
-        {errorMessage}
-      </Typography>
-    );
-  }
+  const { healthChecks } = props;
```
Comment (goeson00) [inline cogboard-webapp/src/components/widgets/types/AemHealthcheckWidget/index.js]: Destructure props in parameters parentheses.

File: cogboard-webapp/src/components/widgets/types/BambooPlanWidget.js [hunk 4]
```diff
@@ -2,18 +2,9 @@ import React from 'react';
...
-  const { url, number, lifeCycleState, errorMessage } = props;
-
-  if (errorMessage) {
-    return (
-      <Typography variant="h5">
-        {errorMessage}
-      </Typography>
-    );
-  }
+  const { url, number, lifeCycleState } = props;
```
Comment (goeson00) [inline cogboard-webapp/src/components/widgets/types/BambooPlanWidget.js]: The same.

File: cogboard-webapp/src/components/widgets/types/IframeEmbedWidget/index.js [hunk 5]
```diff
@@ -1,8 +1,9 @@
 import React from 'react';
-import {StyledIframe} from "../../../styled";
-import {string} from "prop-types";
-import {Typography} from "@material-ui/core/index";
-import {attachHttp} from "./helpers";
+import { string } from "prop-types";
```
Comment (goeson00) [inline cogboard-webapp/src/components/widgets/types/IframeEmbedWidget/index.js]: Empty line, attachHtml is not a lib.

File: cogboard-webapp/src/components/Widget/styled.js (hunk 1 above)
Comment (szymon-owczarzak) [reply cogboard-webapp/src/components/Widget/styled.js]: done

File: cogboard-webapp/src/components/Widget/index.js (hunk 2 above)
Comment (szymon-owczarzak) [reply cogboard-webapp/src/components/Widget/index.js]: done

**Pull request merged**